### PR TITLE
Support gatling v3.5 simulation.log

### PR DESCRIPTION
**Pull request by thyzzv**
The simulation log seems to have changed some where between version 3.2 and 3.5.
A new parser is added to support version 3.5

### CODE REVIEW

File: src/main/java/org/nuxeo/tools/gatling/report/ParserFactory.java [hunk 1]
```diff
@@ -35,7 +35,10 @@ protected static SimulationParser getVersionSpecificParser(File file, Float apde
         // System.out.println(header.size() + " " + header);
         if (header.size() == 6) {
             String version = header.get(5);
-            if (version.matches("3\\.[2-9].*")) {
+            if (version.matches("3\\.[5-9].*")) {
+                return new SimulationParserV35(file, apdexT);
+            }
+            if (version.matches("3\\.[2-4].*")) {
```
Comment (aakankshap) [inline src/main/java/org/nuxeo/tools/gatling/report/ParserFactory.java:41]: Hi @thyzzv , I'm also looking to get support but for gatling 3.4 and the changes to the simulation log are from 3.4 onwards. Could you change this to have 3.4+ in one case? Thanks!

Comment (aakankshap) [reply src/main/java/org/nuxeo/tools/gatling/report/ParserFactory.java:41]: Actually I opened a separate PR for 3.4 as there seem to be other format changes between 3.4 and 3.5. Please ignore the prior comment.

**Comment by snazy** [comment]
In case you're interested: I've pushed a rebased version of your PR to [my fork](https://github.com/snazy/gatling-report/tree/gatling-3.5-rebased)

**Comment by solyard** [comment]
> In case you're interested: I've pushed a rebased version of your PR to [my fork](https://github.com/snazy/gatling-report/tree/gatling-3.5-rebased)

You just saved my life mate. Thanks!

**Pull request merged**